applications/tari_validator_node/src/base_layer_scanner.rs

```diff
@@ -284,7 +285,23 @@ impl BaseLayerScanner {
                 })?;
                 match sidechain_feature {
                     SideChainFeature::ValidatorNodeRegistration(reg) => {
-                        self.register_validator_node_registration(current_height, reg).await?;
+                        let validator_node_timeout = self
```

Comment:
Here we are acknowledging that we received a validator node registration on chain

The `validator_node_registration_expiry` is related to 
- when we should reregister (I'd say the logic for that belongs applications/tari_validator_node/src/registration.rs)
- whether other nodes are still registered. I don't think this is implemented so we can leave that out for now in this PR because we need a good strategy to manage other peers

Reply:
Thanks for the explanation. From the comment, I understood that `validator_node_registration_expiry` alone is not enough to execute a re-registration of the current VN. What should be the re-registration criteria for the current VN on the registration status of other VN's ?